bugwarrior/services/redmine.py

```diff
@@ -31,16 +32,21 @@ def __init__(self, url, key, auth, issue_limit, verify_ssl):
         self.issue_limit = issue_limit
         self.verify_ssl = verify_ssl
 
-    def find_issues(self, issue_limit, only_if_assigned=False):
+    def find_issues(self, issue_limit, query, only_if_assigned=False):
         args = {}
         # TODO: if issue_limit is greater than 100, implement pagination to return all issues.
         # Leave the implementation of this to the unlucky soul with >100 issues assigned to them.
-        if issue_limit is not None:
-            args["limit"] = issue_limit
+        url = "/issues.json"
+        if query is not None:
```

Comment:
Isn't the default value an empty string, rather than None?

Reply:
You're right ! Solved it